orchestra/bots/sanitybot.py

```diff
@@ -0,0 +1,92 @@
+from django.db.models import Max
+from django.utils import timezone
+from pydoc import locate
+
+from orchestra.core.errors import SanityBotError
+from orchestra.models import Project
+from orchestra.models import SanityCheck
+from orchestra.models import WorkflowVersion
+from orchestra.utils.notifications import message_experts_slack_group
+from orchestra.utils.project_properties import incomplete_projects
+
+
+def _workflow_versions_with_sanity_checks():
+    # TODO(marcua): make more specific.
```

Comment:
This doesn't incur serious performance issues in practice, so I'm tempted to leave as-is :)